README.md

```diff
@@ -38,765 +56,196 @@ Given that Webpacker already provides React integration, why would you use "Reac
+Please email me for a free half-hour project consultation, on anything from React on Rails to any aspect of web development.
 
-**_comments.html.erb**
-```erb
-<%= react_component("CommentsApp") %>
-```
+### Testimonials
 
-**_blogs.html.erb**
-```erb
-<%= react_component("BlogsApp") %>
-```
+Check our [testimonials](docs/testimonials.md).
 
-*Note:* You will not be doing any partial updates to the Redux store when loading a new page. When the page content loads, React on Rails will rehydrate a new version of the store with whatever props are placed on the page.
+## Additional Documentation
 
-## ReactOnRails JavaScript API
-See [ReactOnRails JavaScript API](docs/api/javascript-api.md).
+Please check our [docs index](./SUMMARY.md) for links to the files within the docs directory.
 
-#### Using Rails built-in CSRF protection in JavaScript
+## Community Resources
 
-Rails has built-in protection for Cross-Site Request Forgery (CSRF), see [Rails Documentation](http://guides.rubyonrails.org/security.html#cross-site-request-forgery-csrf). To nicely utilize this feature in JavaScript requests, React on Rails provides two helpers that can be used as following for POST, PUT or DELETE requests:
+Please [**click to subscribe**](https://app.mailerlite.com/webforms/landing/l1d9x5) to keep in touch with Justin Gordon and [ShakaCode](http://www.shakacode.com/). I intend to send announcements of new releases of React on Rails and of our latest [blog articles](https://blog.shakacode.com) and tutorials. Subscribers will also have access to **exclusive content**, including tips and examples.
 
-```js
-import ReactOnRails from 'react-on-rails';
+[![2017-01-31_14-16-56](https://cloud.githubusercontent.com/assets/1118459/22490211/f7a70418-e7bf-11e6-9bef-b3ccd715dbf8.png)](https://app.mailerlite.com/webforms/landing/l1d9x5)
 
-// reads from DOM csrf token generated by Rails in <%= csrf_meta_tags %>
-csrfToken = ReactOnRails.authenticityToken();
+- **Slack Room**: [Contact us](mailto:contact@shakacode.com) for an invite to the ShakaCode Slack room! Let us know if you want to contribute.
+- **[forum.shakacode.com](https://forum.shakacode.com)**: Post your questions
+- **[@railsonmaui on Twitter](https://twitter.com/railsonmaui)**
+- For a live, [open source](https://github.com/shakacode/react-webpack-rails-tutorial), example of this gem, see [www.reactrails.com](http://www.reactrails.com).
+- See [Projects](PROJECTS.md) using and [KUDOS](./KUDOS.md) for React on Rails. Please submit yours! Please edit either page or [email us](mailto:contact@shakacode.com) and we'll add your info. We also **love stars** as it helps us attract new users and contributors.
+- *See [NEWS.md](NEWS.md) for more notes over time.*
 
-// compose Rails specific request header as following { X-CSRF-Token: csrfToken, X-Requested-With: XMLHttpRequest }
-header = ReactOnRails.authenticityHeaders(otherHeader);
-```
+##Contributing
```

Comment:
We should add a [space between header and markup](https://github.com/markdownlint/markdownlint/blob/e3dbc768e1608da5baab0a5896c54e840ff59110/docs/RULES.md#md018---no-space-after-hash-on-atx-style-header).